Source/WebCore/html/HTMLVideoElement.cpp

```diff
@@ -319,7 +319,7 @@ bool HTMLVideoElement::shouldDisplayPosterImage() const
 
 void HTMLVideoElement::mediaPlayerFirstVideoFrameAvailable()
 {
-    ALWAYS_LOG(LOGIDENTIFIER, "m_showPoster = ", showPosterFlag());
+    HTMLVIDEOELEMENT_RELEASE_LOG(HTMLVIDEOELEMENT_MEDIAPLAYERFIRSTVIDEOFRAMEAVAILABLE, showPosterFlag());
```

Comment:
This is rather verbose, is there a way we could make things shorter somehow?
Can we use `__FILE__` and `__func__`?

Also we have twice the `HTMLVIDEOELEMENT_`, maybe we could have it only once?

Reply:
In the latest patch, I have removed the `HTMLVIDEOELEMENT_` part of the format string identifier.

It may not be feasible to use `__func__` in this context, since it cannot be used in a log format string because it is not a string literal. The improved efficiency in this patch is based on having as much as possible of the logs in the log format strings, since the format string is not sent over IPC.

Thanks for reviewing!